nautobot/core/templates/admin/config/config.html

```diff
@@ -2,6 +2,14 @@
 {% load admin_list static i18n %}
 {% load helpers %}
 
+{% block breadcrumbs %}
+  <ul class="breadcrumb">
+    <li><a href="{% url 'admin:index' %}">{% trans 'Admin Home' %}</a></li>
+    <li><a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+    <li>{{ opts.verbose_name_plural|capfirst }}</li>
+  </ul>
+{% endblock %}
+
```

Comment:
Can you provide a bit of context here? Why does this page need its own custom breadcrumbs code, is it not using the core template?

Do other "object list" pages (e.g. `admin/extras/jobresult/`) also need custom breadcrumbs or do they "just work" with the `app_index.html` changes above?

Reply:
We don't really have a standard core template for breadcrumbs. Outside of Admin, the only place we have in core is `object_detail.html`.  The object list page that you are referring to is covered under  `core/templates/admin/change_list.html`.  For the Admin site reskin, it looks like the app_index.html was just left off.  This is why the `admin/extras/jobresult` and `admin/users/user` had breadcrumbs, but the headings such as `admin/users` or `admin/extras` did not.  

`admin/constance/config` was the odd man out, due to the view in django-constance.  

Admittedly, there are some issues I have with this.  The object_detail.html has the breadcrumbs wrapped in an ordered list block (`<ol></ol>`), while all the overrides that were put in the Admin reskin `change_form.html`, `change_list.html`, `delete_confirmation.html` and `index.html` all use unorderd list(`<ul></ul>`).

Reply:
Bootstrap's own [breadcrumbs example](https://getbootstrap.com/docs/3.3/components/#breadcrumbs) uses `<ol>` so we probably should do the same for consistency. It is a list in which order is significant, after all.